Projects/Server/GameCounter.h

```diff
@@ -0,0 +1,43 @@
+#ifndef GAMECOUNTER_H
+#define GAMECOUNTER_H
+
+#include "State.h"
+#include "GameHandler.h"
+
+#include <map>
+
+
+
+class GameCounter {
```

Comment:
At least in common (Core/Server) code please follow the established coding styles.
E.g. starting curly brackets on new line.

Reply:
Any such deviations which I could manage to find have been rectified.